.github/CODEOWNERS

```
@@ -0,0 +1 @@
+**/docs/**.md @margaretkennedy
```

You need the `**` for the directory, then `*.md` for the file.
```suggestion
**/docs/**/*.md @margaretkennedy
```